### PR TITLE
16 as a user i want to create a chatroom so that i can chat other users

### LINKED ISSUE #16: As a user, I want to create a chatroom so that I can chat other users.

### PR DESCRIPTION
**Pull request by hale2ian**

### CODE REVIEW

File: .idea/.gitignore
Comment (marktmng) [inline .idea/.gitignore:1]: Thank you for editing.

File: .idea/inspectionProfiles/Project_Default.xml
Comment (marktmng) [inline .idea/inspectionProfiles/Project_Default.xml:1]: Thank you for your hard work.

File: .idea/.gitignore
Comment (hale2ian) [reply .idea/.gitignore:1]: you are welcome. :)

File: .idea/inspectionProfiles/Project_Default.xml
Comment (hale2ian) [reply .idea/inspectionProfiles/Project_Default.xml:1]: you are welcome. :)

**Review by marktmng — APPROVED** [review]

**Pull request merged**